assets/tests/components/mapPage/vehiclePropertiesCard.test.tsx

```diff
@@ -176,8 +197,8 @@ describe("<VehiclePropertiesCard/>", () => {
         const vehicle = vehicleFactory.build()
         const intersection = "intersection ave @ street"
         ;(useNearestIntersection as jest.Mock)
-          .mockReturnValueOnce(undefined)
-          .mockReturnValueOnce(intersection)
+          .mockReturnValueOnce({ is_loading: true })
```

Comment:
question(non-blocking): Non-blocking since this seems to replicate the existing setup of the test, but looks like the part where it tests the loading state text is commented out

Reply:
Thanks for catching! I'll uncomment it.